.eslintignore

```diff
@@ -1,2 +1,3 @@
+!.*
```

Comment:
By default, ESLint ignores everything starting with a dot. I don't see any reason to, and it breaks the pre-commit hook, so have added this to unignore such files.